src/main/kotlin/org/jitsi/videobridge/octo/config/OctoConfig.kt

```diff
@@ -38,6 +38,60 @@ class OctoConfig {
                     null
                 }
             }
+
+            class BindAddressProperty : LegacyFallbackConfigProperty<String>(
+                String::class,
+                "org.jitsi.videobridge.octo.BIND_ADDRESS",
+                "videobridge.octo.bind-address",
+                readOnce = true
+            )
+
+            private val bindAddressProp = BindAddressProperty()
+
+            @JvmStatic
+            fun bindAddress(): String? {
```

Comment:
Since we're updating the config, can we add a `videobridge.octo.enabled` flag, and make the bind address required?